pkg/kubelet/network/hostport/fake_iptables.go

```diff
@@ -228,36 +228,27 @@ func saveChain(chain *fakeChain, data *bytes.Buffer) {
 }
 
 func (f *fakeIPTables) Save(tableName utiliptables.Table) ([]byte, error) {
```

Comment:
We are still using it in few places in kubelet. Though I'm happy to send a PR migrating it to SaveInto (should be pretty simple). But I would prefer to do it in a separate one. Will send one on Monday.